concepts/05 Widgets/DateBox/00 Getting Started with DateBox/25 Disable Specific Dates.md

```diff
@@ -0,0 +1,241 @@
+To prevent users from setting specific dates, use the [disabledDates](/Documentation/ApiReference/UI_Widgets/dxDateBox/Configuration/#disabledDates) option. It can be set to a function or an array of Date values. In this tutorial, we implement the function, and it disables weekend days and the US bank holidays.
```

Comment:
```suggestion
To prevent users from setting specific dates, use the [disabledDates](/Documentation/ApiReference/UI_Widgets/dxDateBox/Configuration/#disabledDates) option. It can be set to a function or an array of Date values. In this tutorial, we implement a function that disables weekends and US bank holidays.
```